strings/java_strings_intruduction/Solution.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+import java.lang.String;
+
+public class Solution {
+  
+  private static int lengthSum(String firstStringToSum, String secondStringToSum) {
+    return (firstStringToSum.length() + secondStringToSum.length());
+  }
+  
+  private static String compareStringLexicographically(String firstStringToCompare, String secondStringToCompare) {
+    if (firstStringToCompare.compareTo(secondStringToCompare) > 0) {
+      return "Yes";
+    } else {
+        return "No";
+      }
+  }
+  
+  private static String capitalizeFirstLetter(String firstStringToCapitalize, String secondStringToCapitalize) {
```

Comment:
To verbose :)
Also make it a function of one String, because the actions of input Strings are independent besides concatenating in the end, which can be done outside of the function. 
Generally a function should only take two arguments if it needs both of them to calculate some value. For example: `multiply(int a, int b)`, `add(int first, int second)`, `findSubstrings(String pattern, String line)`.
A function shouldn't do independent things on inputs. Example of bad function signatures: `capitalizeStringAndAddSquaredValue(String stringWithLowerCaseSymbols, int valueToBeSquaredAndConcatenated)`.
Also a function should have a clear goal described without "and". For example "this function capitalizes input string", "this function multiplies input numbers". And not "this function capitalizes input strings AND concatenates them", "this function fetches data AND filters out bad values".
Of course in more complicated situations it's harder to define boundaries, but for such simple things as this function it's clear that the function is not following the rule to have a single responsibility.